l10n_fi_business_code/models/res_partner.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Oy Tawasta OS Technologies Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    business_id = fields.Char(
+        string='Business ID',
+        compute=lambda s: s._compute_identification(
```

Comment:
Not sure if it's a big problem, but I'll add a note here anyway: using a lambda here means that anyone who wants to extend this model cannot directly extend the computation method of the field `business_id` in an Odoo-like fashion: they either have to do some pretty gnarly monkeypatching, or extend `_compute_identification` and trust or ensure that their changes only apply to the computation of this field. I think it would be a little better if we just go the normal route and define real computation methods and then refer to them by string in the field definition.

Reply:
I think this is a valid concern, but using lambda function is the suggested style on the partner_identification module:
https://github.com/OCA/partner-contact/blob/10.0/partner_identification/models/res_partner.py#L35

I'd guess this is so we can pass the desired category code for compute functions easily.
Also, as said, these all refer to an actual `_helper_method()` that can be extended when necessary.

This style is also used in
https://github.com/OCA/partner-contact/blob/10.0/partner_coc/models/res_partner.py#L7

I support changing this, but I would rather not to do this in this PR.